src/main/kotlin/gg/octave/bot/commands/music/NowPlaying.kt

```diff
@@ -43,8 +43,15 @@ class NowPlaying : MusicCog {
                 true
             )
             addBlankField(true)
-            addField("Repeating", manager.scheduler.repeatOption.name.toLowerCase().capitalize(), true)
+
             addField("Volume", "${manager.player.volume}%", true)
+            addBlankField(true)
+            addBlankField(true)
+
+            addField("Repeating", manager.scheduler.repeatOption.name.toLowerCase().capitalize(), true)
+            addField("Shuffle", manager.scheduler.autoShuffle.name.toLowerCase().capitalize(), true)
+            addBlankField(true)
+
             addField("Bass Boost", manager.dspFilter.bassBoost.name.toLowerCase().capitalize(), true)
```

Comment:
Replace addBlankField(true) with bass boost. (Format is broken on mobile.)
```suggestion
            addField("Bass Boost", manager.dspFilter.bassBoost.name.toLowerCase().capitalize(), true)
```